tools/windows/install-help/cal/FinalizeInstall.cpp

```diff
@@ -293,18 +293,6 @@ UINT doFinalizeInstall(CustomActionData &data)
     if (!ddServiceExists)
     {
         WcaLog(LOGMSG_STANDARD, "attempting to install services");
-        if (!passToUse)
-        {
-            if (!data.value(propertyDDAgentUserPassword, providedPassword))
-            {
-                // given all the error conditions checked above, this should *never*
-                // happen.  But we'll check anyway
-                WcaLog(LOGMSG_STANDARD, "Don't have password to register service");
-                er = ERROR_INSTALL_FAILURE;
-                goto LExit;
-            }
-            passToUse = providedPassword.c_str();
-        }
```

Comment:
Why is this section is completely removed while in cainstall.cpp this line is left
if (!data.present(propertyDDAgentUserPassword))